recipes/torchtext/meta.yaml

```diff
@@ -0,0 +1,126 @@
+{% set name = "torchtext" %}
+{% set version = "0.14.1" %}
+{% set number = 0 %}
+
+{% if cuda_compiler_version != "None" %}
+{% set number = number + 200 %}
+{% endif %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pytorch/text/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: fd1ef3da7d9c20408c740f7dc7d02ad52a6048b46368355a1a7326d3bc4f2e63
+  # patches:
+  #   - patches/fix_dispatch_apply_auto.patch
+  #   - patches/osx-map_anonymous.patch
+  #   # 2022/07 hmaarrfk
+  #   # We've patch out the bug in GCC that makes it incompatible
+  #   # https://github.com/conda-forge/ctng-compilers-feedstock/blob/7ea10c2526445beb9faff23efcc6bc40fc0079b5/recipe/meta.yaml#L23=
+  #   - patches/remove_gcc_maximum_requirements.patch 
+
+build:
+  number: {{ number }}
+  skip: true  # [win]
+  skip: true  # [cuda_compiler_version == "10.2"]
+  skip: true  # [cuda_compiler_version == "11.0"]
+  skip: true  # [cuda_compiler_version == "11.1"]
+
+outputs:
+  - name: torchtext
+    build:
+      string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+      string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                                # [cuda_compiler_version == "None"]
+      detect_binary_files_with_prefix: false
+      # run_exports:
+      #   - {{ pin_subpackage('pytorch', max_pin='x.x') }}
+    script: build_pytorch.sh  # [not win]
+    script: bld_pytorch.bat   # [win]
```

Comment:
Maybe update the script names to indicate they're building `torchtext` rather than `pytorch`?

Reply:
@dhirschfeld I have changed the build script names to `torchtext` as suggested.